client/src/modules/reports/generate/configurable_analysis_report/configurable_analysis_report.html

```diff
@@ -21,54 +21,64 @@ <h3 class="text-capitalize" translate>REPORT.CONFIGURABLE_ANALYSIS_REPORT.TITLE<
 
         <div class="panel-body">
           <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
-            <bh-fiscal-period-select
-              on-select-fiscal-callback="ReportConfigCtrl.onSelectFiscal(fiscal)"
-              on-select-period-from-callback="ReportConfigCtrl.onSelectPeriodFrom(period)"
-              on-select-period-to-callback="ReportConfigCtrl.onSelectPeriodTo(period)">
-            </bh-fiscal-period-select>
 
-            <hr>
-            <bh-multiple-cashbox-select
-              cashbox-ids="ReportConfigCtrl.reportDetails.cashboxes"
-              on-change="ReportConfigCtrl.onSelectCashboxes(cashboxes)"
-              required="true">
-            </bh-multiple-cashbox-select>
-
-            <hr>
-            <div class="checkbox">
-              <label>
-                <input type="checkbox" ng-model="ReportConfigCtrl.reportDetails.hide_account_details" ng-true-value="0" ng-false-value="1">
-                <span translate>REPORT.CONFIGURABLE_ANALYSIS_REPORT.HIDE_ACCOUNT_DETAILS</span>
-              </label>
-            </div>
-            <div class="checkbox">
-              <label>
-                <input type="checkbox" ng-model="ReportConfigCtrl.reportDetails.hide_details_types" ng-true-value="0" ng-false-value="1">
-                <span translate>REPORT.CONFIGURABLE_ANALYSIS_REPORT.HIDE_DETAILS_TYPES</span>
-              </label>
+            <div ng-if="ReportConfigCtrl.configurationsAreMissing" style="margin-bottom: 1em;">
+              <span class="label label-warning text-capitalize" style="padding: 6px;">
+                <i class="fa fa-exclamation-triangle"></i>
+                <span style="font-size: 1.75em;" translate> FORM.INFO.THE_NECESSARY_CONFIG_MISSING </span>
```

Comment:
Let's make this into a proper `alert alert-danger` and change the text a bit.

For example:
![image](https://user-images.githubusercontent.com/896472/170260455-0e45dc79-51b3-4b1a-9d32-452f02418f20.png)


We should say something like
"Warning"
"The necessary configuration elements are missing.  If you would like to set them now, please follow this link."

And link to the configuration analysis tools page:

![image](https://user-images.githubusercontent.com/896472/170258579-4931163f-120a-45fd-912f-f8a3368da9ac.png)

Reply:
What about just hiding all form elements if it isn't configured?

Reply:
> Let's make this into a proper `alert alert-danger` and change the text a bit.

Ok.

> We should say something like "Warning" "The necessary configuration elements are missing. If you would like to set them now, please follow this link."
> 
> And link to the configuration analysis tools page:

Good idea.  Will do.

Reply:
Done